libs/ui/packages/react/src/pre-ldls/components/VirtualList/VirtualList.tsx

```diff
@@ -1,27 +1,28 @@
-import React from "react";
+import React, { ReactNode } from "react";
 import { FixedSizeList as List } from "react-window";
 
 export const VirtualList = ({
-  children,
+  count,
   itemHeight,
   maxHeight,
+  renderRow,
 }: {
-  children: React.ReactNode[];
+  count: number;
   itemHeight: number;
   maxHeight: number;
+  renderRow: (index: number) => ReactNode;
 }) => {
-  const childrenLength = Array.isArray(children) ? children.length : 0;
-  const minHeight = Math.min(childrenLength * itemHeight, maxHeight);
+  const minHeight = Math.min(count * itemHeight, maxHeight);
```

Comment:
I don't think we want to pass the height manually. Could you wrap the list in the `AutoSizer` component like in [CurrencyList.tsx](https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-desktop/src/renderer/drawers/DataSelector/CurrencyList.tsx).

Also was there a reason we needed this minHeight what would have happen if e.g there was no items in the list and we still passed a maxHeight as the list height ?